test/unit/helpers/DismissHelper.ts

```diff
@@ -14,11 +14,15 @@ import { DismissHelper } from "../../../src/helpers/DismissHelper";
 import { SlidedownManager } from "../../../src/managers/slidedownManager/SlidedownManager";
 import TimedLocalStorage from "../../../src/modules/TimedLocalStorage";
 import { DismissTimeKey } from "../../../src/models/Dismiss";
+import Slidedown from "../../../src/slidedown/Slidedown";
+import { AutoPromptOptions } from "../../../src/managers/PromptsManager";
 
 const sinonSandbox: SinonSandbox = sinon.sandbox.create();
 const testHelper = new SlidedownPromptingTestHelper(sinonSandbox);
-const minimalPushSlidedownOptions = testHelper.getMinimalPushSlidedownOptions();
-const minimalSmsAndEmailOptions = testHelper.getMinimalSmsAndEmailOptions();
+
+const minimalPushSlidedownOptions     = testHelper.getMinimalPushSlidedownOptions();
+const minimalSmsAndEmailOptions       = testHelper.getMinimalSmsAndEmailOptions();
+const minimalCategorySlidedownOptions = testHelper.getMinimalCategorySlidedownOptions();
```

Comment:
nit, I think we should stop aligning on `=` as this creates a larger diff. This means a harder to review PR and higher chance of merge conflicts. We can discuss offline as well.